recipe/meta.yaml

```diff
@@ -38,8 +45,8 @@ requirements:
     - cmake         # [not osx]
     - cmake 3.20.*  # [osx]
     - perl
-    - llvm-openmp  # [osx]
-    - libgomp      # [linux and not aarch64]
+    - llvm-openmp  # [osx and build_platform != "linux-64"]
```

Comment:
Why is this needed?

Reply:
I'm not sure, I can remove it and see if it changes anything. That change came from here:
https://github.com/conda-forge/ambertools-feedstock/commit/e260850480f6c1ba0696c31a68ed94802715ad99